api/http/handler/endpointgroups/endpoints.go

```diff
@@ -0,0 +1,32 @@
+package endpointgroups
+
+import portainer "github.com/portainer/portainer/api"
+
+func (handler *Handler) updateEndpointRelations(endpoint portainer.Endpoint, endpointGroup portainer.EndpointGroup) error {
```

Comment:
Any particular reason why are we not using pointers here? See my point above but as a rule of thumb use pointers when you can.

Reply:
no reason I wasn't sure about that. went both ways and in the end, changed to no pointers. I'll update the code.